Modules/Sources/NineAnimatorNativeParsers/MediaParsers/StreamSBParser.swift

```diff
@@ -23,58 +23,91 @@ import NineAnimatorCommon
 
 class StreamSBParser: VideoProviderParser {
     var aliases: [String] { [] }
-    
-    private let apiPath = URL(string: "https://streamsb.net/play")!
-    
-    private let videoSourceRegex = try! NSRegularExpression(
-        pattern: #"file:"([^"]+)"#,
-        options: .caseInsensitive
-    )
-    
-    private let embedIDRegex = try! NSRegularExpression(
-        pattern: #"(?<=embed-).+?(?=\.)"#,
-        options: .caseInsensitive
-    )
-    
-    private func retrieveId(from path: String) -> String? {
-        self.embedIDRegex
-            .firstMatch(in: path)?
-            .first
+
+    private struct StreamSBAPIResponse: Codable {
+        let streamData: StreamData
+        let statusCode: Int
+        let userData: UserData
+
+        enum CodingKeys: String, CodingKey {
```

Comment:
JSONDecoder has a built-in [`.convertFromSnakeCase`](https://developer.apple.com/documentation/foundation/jsondecoder/keydecodingstrategy/convertfromsnakecase) that you can use to avoid specifying custom CodingKeys.